source/server/http/admin.h

```diff
@@ -250,9 +250,9 @@ class PrometheusStatsFormatter {
    * Extracts counters and gauges and relevant tags, appending them to
    * the response buffer after sanitizing the metric / label names.
    */
-  static void statsAsPrometheus(const std::list<Stats::CounterSharedPtr>& counters,
-                                const std::list<Stats::GaugeSharedPtr>& gauges,
-                                Buffer::Instance& response);
+  static uint64_t statsAsPrometheus(const std::list<Stats::CounterSharedPtr>& counters,
```

Comment:
nit: please update the method documentation with `@return uint64_t explanation of return value.`